mobile/library/cc/engine_builder.h

```diff
@@ -62,6 +62,11 @@ class XdsBuilder final {
   // connection. If no root certs are specified, the operating system defaults are used.
   XdsBuilder& setSslRootCerts(std::string root_certs);
 
+  // Sets the SNI (https://datatracker.ietf.org/doc/html/rfc6066#section-3) on the TLS handshake
+  // and the authority HTTP header. If not set, the SNI is set by default to the xDS server address
+  // and the authority HTTP header is not set.
+  XdsBuilder& setSNI(std::string sni);
```

Comment:
I think this should be setSni() per the style guide

"For names written in mixed case (also sometimes referred to as "[camel case](https://en.wikipedia.org/wiki/Camel_case)" or "[Pascal case](https://en.wiktionary.org/wiki/Pascal_case)"), in which the first letter of each word is capitalized, prefer to capitalize abbreviations as single words, e.g., StartRpc() rather than StartRPC()."

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
makes sense and i was debating that, but forgot about the actual style guide recommendation on this topic, thanks for pointing it out!  i just updated the method name in all the files, as the Java style guide says similar (https://google.github.io/styleguide/javaguide.html#s5-naming).